### DISCUSSION TITLE
need help with my door sensors

### DISCUSSION THREAD

**Discussion by miloit** [zwave-js/zwave-js-ui#3574] [Q&A] [answered]
I have 2 door sensors in my zwave network

The fiirst node id 14 is showing evrything and also the basic topic:
![Bildschirmfoto vom 2024-02-01 17-44-01](https://github.com/zwave-js/zwave-js-ui/assets/13338603/0e507b47-c489-4e04-86be-b471d38cf167)

But node id 15 is not showing any basic topi
![Bildschirmfoto vom 2024-02-01 17-43-45](https://github.com/zwave-js/zwave-js-ui/assets/13338603/119f2fa5-8b09-48c6-a6ec-6264add2d814)

How can i get this back?

**Comment by robertsLando** [comment]
Try to do a re-interview of your device

**Reply by miloit** [reply]
Tried it already several times but no success

used this here:

![Bildschirmfoto vom 2024-02-01 18-17-40](https://github.com/zwave-js/zwave-js-ui/assets/13338603/f3b13dc9-98fd-48e6-8302-547331700cc0)

**Reply by robertsLando** [reply]
Please make a [driver log](https://zwave-js.github.io/zwave-js-ui/#/troubleshooting/generating-logs?id=driver-logs) of a re-interview, loglevel `debug` and attach it here as a file (drag & drop into the text field).

cc @AlCalzone

**Reply by miloit** [reply, marked as answer]
I changed battery and reinterviewed it now working...thanks for your help
